CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related
+suggestions :mag_right:. We distinguish between different [different types of changes](#types-of-changes):
+Maintenance, Enhancements, Features and Bugs. This section only concerns the first three of these. If you want to learn
+about how to report bugs, [this](#reporting-bugs) might help you.
+
+Before creating a suggestion, please check Check if there's already an [issue](https://github.com/sakuli/sakuli/issues)
+addressing your suggestion. When you are creating an enhancement/feature suggestion, please fill in
+the [enhancement](https://github.com/sakuli/sakuli/issues/new?template=enhancement.md) or
+[feature template](https://github.com/sakuli/sakuli/issues/new?template=feature.md) and ensure, that the 
+[definition of ready](#definition-of-ready) is fulfilled.
+
+### Contributions
+Please feel free to begin with any issue you would like to work on. It might be helpful to get some orientation by having
+a look at our [types of changes](#types-of-changes) and the estimated requirements linked to them.  
+
+>**Important Note**: If you'd like to contribute a bug fix, please ensure to branch from `develop` instead of master.
+
+We also highly appreciate contributions to milestones and we love to work closely with the community.
+So if you'd like to work on an issue that is scheduled for a release, please make sure to stay in close contact with the
+maintainers and make sure that it's possible for you to propose a [pull request](#pull-requests) before the planned
+release date of the milestone, if specified.  
+
+#### Unsure where to begin contributing to Sakuli?
+You can start by looking for issues labeled with [good first issue](https://github.com/sakuli/sakuli/labels/good%20first%20issue) which should only require a few lines of code, and
+a test or two. 
+
+### Pull Requests
+Pull requests are required for every contribution to Sakuli including contributions from maintainers and administrators.
+Before you propose a pull request, please ensure, that the [definition of done](#definition-of-done) is fulfilled.
+Please ensure that every pull request is linked to an issue. If you open a pull request without an existing issue, please 
+open an issue as well, to point out the requirements for the change. That helps us to separate the functional discussion 
+(issue) from the technical discussion (pull-request).
+
+This helps us to reach several goals:
+* Maintain the high quality standards of Sakuli
+* Make changes transparent to the community
+* Discuss changes to ensure that the best possible solution will make it into the next release
+* Enable a sustainable system for Sakuli maintainers to review contributions
+
+>**Important Note**: If you'd like to propose a pull request for a bug fix, please ensure that your base branch is
+`develop` instead of master and that the target branch for the pull request is `develop` as well .
```

Comment:
```suggestion
>**Important Note**: If you'd like to propose a pull request for a bug fix, please make sure that your base branch is
`develop` instead of master and that the target branch for the pull request is `develop` as well.
```